src/main/java/uk/gov/digital/ho/hocs/client/document/DocumentClient.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.digital.ho.hocs.client.document;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import uk.gov.digital.ho.hocs.application.RestClient;
+import uk.gov.digital.ho.hocs.client.document.dto.CreateDocumentRequest;
+
+import java.util.UUID;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static uk.gov.digital.ho.hocs.application.LogEvent.DOCUMENT_CLIENT_CREATE_SUCCESS;
+import static uk.gov.digital.ho.hocs.application.LogEvent.EVENT;
+
+@Slf4j
+@Component
```

Comment:
Can this have `@Profile("migration")` as it's only used there, otherwise we need to add the secret to the case creator helm chart too

Reply:
I tried putting in that annotation, but it broke a load of tests, non-migration related. 
I've made this [PR](https://github.com/UKHomeOffice/hocs-helm-charts/pull/88) for the helm chart changes as mentioned in our other comment.  Haven't worked with helm charts before, is this what you were expecting?

Reply:
I've got it working with the `@Profile("migration")`, so don't need to add the secret to case-creator, just case-migrator for the helm-charts now.